src/apolo_app_types/protocols/spark_job.py

```diff
@@ -24,10 +24,8 @@ class SparkApplicationType(StrEnum):
 class DriverConfig(AbstractAppFieldType):
     preset: Preset = Field(
         ...,
-        json_schema_extra=SchemaExtraMetadata(
-            title="Driver Preset",
-            description="Preset configuration to be used by the driver",
-        ).as_json_schema_extra(),
+        title="Driver Preset",
```

Comment:
so, we go with json_schema_extra?

Reply:
Yes, was before our discussion, I will do changes this week in separate PR